add_to_app/flutter_module_using_plugin/lib/cell.dart

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:sensors/sensors.dart';
+
+// This is on alternate entrypoint for this module to display Flutter UI in
+// a (multi-)view integration scenario.
+void main() {
+  runApp(Cell());
+}
+
+class Cell extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => new _CellState();
+}
+
+class _CellState extends State<Cell> with WidgetsBindingObserver {
+  static const double gravity = 9.81;
+  static final AccelerometerEvent defaultPosition = AccelerometerEvent(0, 0, 0);
+
+  int cellNumber = 0;
+  Random randomColor;
+  AppLifecycleState appLifecycleState;
+
+  @override
+  void initState() {
+    final channel = MethodChannel('dev.flutter.example/cell');
+    channel.setMethodCallHandler((MethodCall call) async {
+      if (call.method == 'setCellNumber') {
+        setState(() {
+          cellNumber = call.arguments as int;
+          randomColor = Random(cellNumber);
+        });
+      }
+    });
+    // Keep track of what the current platform lifecycle state is.
+    WidgetsBinding.instance.addObserver(this);
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    WidgetsBinding.instance.removeObserver(this);
+    super.dispose();
+  }
+
+  @override
+  void didChangeAppLifecycleState(AppLifecycleState state) {
+    setState(() { appLifecycleState = state; });
+  }
+
+  // Show a random bright color.
+  Color randomLightColor() {
+    if (randomColor == null) {
+      randomColor = Random(cellNumber);
+    }
+    return Color.fromARGB(255, randomColor.nextInt(50) + 205,
+        randomColor.nextInt(50) + 205, randomColor.nextInt(50) + 205);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      // The Flutter cells will be noticeably different (due to background color
+      // and the Flutter logo). The banner breaks immersion.
+      debugShowCheckedModeBanner: false,
```

Comment:
It looks weird. Since it's a cell that's interwoven into native. But the background is a lot bigger than the cell, so this banner shows up in the middle of nowhere.

Reply:
Fair point. You're right to take it out.